modules/patient-portal/src/main/frontend/src/patient-portal/QuestionnaireSet.jsx

```diff
@@ -175,6 +175,10 @@ function QuestionnaireSet(props) {
     return subjectData?.[questionnaireId] && !subjectData[questionnaireId].statusFlags?.includes("INCOMPLETE");
   }
 
+  const isFormSubmitted = (questionnaireId) => {
+    return subjectData?.[questionnaireId] && subjectData[questionnaireId].statusFlags?.includes("SUBMITTED");
```

Comment:
This could be simplified as


```suggestion
    return subjectData?.[questionnaireId]?.statusFlags?.includes("SUBMITTED");
```